scapy/layers/l2tp.py

```diff
@@ -16,14 +16,29 @@
 from scapy.layers.inet import UDP
 from scapy.layers.ppp import PPP
 
+header_flags = ['type', 'length', 'res02', 'res03', 'sequence', 'res05',
```

Comment:
Please use a more explicit name starting with a _ so that it’s not exported to the entire scapy module